README.md

```diff
@@ -1,2 +1,18 @@
 # sarathy
 "one with a chariot"
+
+### Toolchain info
+
+### Usage
+```bash
+# run minikube container
+# creds: docker / docker
+mkdir -p /tmp/.sarathy/minikube
+
+docker run -it --rm \
+    --privileged -it -h minikube --name sarathy-minikube \
+    --mount type=bind,source=/tmp/.sarathy/minikube,target=/var/lib/docker \
+    savyasachi9/sarathy:minikube
+
```

Comment:
update this with docker exec cmd 
docker exec -it --user docker sarathy-minikube bash